### DISCUSSION TITLE
Where is the default download path?

### DISCUSSION THREAD

**Discussion by nikmax42** [lennart-rth/Live-Earth-Wallpapers#37] [General]
I couldn't find which directory does Liewa use to download images by default. 

UPD. after looking at the code and run command I found the path:
`/home/username/.local/share/pipx/venvs/liewa/lib/python3.12/site-packages/liewa/liewa_cli/recources/backgroundImage.png`
But that was a bit tricky for me. Please add info about it to readme and console outputs.

**Comment by lennart-rth** [comment]
This will do good as a Discussion entry :)
